internal/sessiontest/helper_dosession_test.go

```diff
@@ -332,6 +339,14 @@ func doSession(
 	serverResult := getSessionResult(t, sesPkg, serv, useJWTs, opts)
 	require.Equal(t, sesPkg.Token, serverResult.Token)
 
+	if opts.enabled(optionExpectSdJwts) {
```

Comment:
Should be a `TODO` with the code completely commented out, to satisfy the linter.